corehq/apps/accounting/async_handlers.py

```diff
@@ -212,9 +213,29 @@ def account_response(self):
     @property
     def plan_version_response(self):
```

Comment:
Do we have tests that cover the result of this function? If not, would it be difficult to add happy path tests for the changes?

Reply:
Thanks Amit, add test in 6d67c8f